README.md

```
@@ -16,19 +16,31 @@
     <a href="https://rudderstack.com/join-rudderstack-slack-community">Community Slack</a>
   </b>
 </p>
 
 ---
 
+> ⚠️ **DEPRECATION WARNING**
+>
+> **This SDK is deprecated and no longer actively maintained.**
+>
+> Please use the newer **Swift-based iOS SDK** instead:
+> - **Repository**: https://github.com/rudderlabs/rudder-sdk-swift
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)